Documentation/dotnet-mage/README.md

```diff
@@ -2,7 +2,7 @@
 
 dotnet-mage (previously also known as Mage.NET) is the open-sourced version of the familiar .NET FX tool Mage.
 
-It is available at Nuget.org. Latest version `https://www.nuget.org/packages/Microsoft.DotNet.Mage/6.0.1`
+It is available at Nuget.org. Latest version `https://www.nuget.org/packages/Microsoft.DotNet.Mage/7.0.0`
```

Comment:
Would it make sense to have an aka.ms link for the latest version, then you never need to update it? I'm not 100% sure if we can use those for OSS, but would simplify things

Reply:
Not sure - I'm going to check. Thanks!